src/stories/3-fields/select/select.stories.tsx

```diff
@@ -24,17 +22,6 @@ export default {
 		...ExcludeReactFormHookProps,
 		...CommonFieldStoryProps("select"),
 		select: { table: { disable: true } },
```

Comment:
disable controls for `multiselect-default` instead of `select`

Reply:
did you mean to comment this in `multi-select.stories.tsx`?